runtime/compiler/control/JITServerCompilationThread.cpp

```diff
@@ -167,7 +189,7 @@ outOfProcessCompilationEnd(
                                                          std::vector<TR_ResolvedJ9Method*>(resolvedMirrorMethodsPersistIPInfo->begin(), resolvedMirrorMethodsPersistIPInfo->end()) :
                                                          std::vector<TR_ResolvedJ9Method*>(),
                                      *entry->_optimizationPlan, serializedRuntimeAssumptions, memoryState,
```

Comment:
I would prefer `memoryState` and `activeThreadState` to be next to each other as parameters.